core/metrics-core-service/src/main/java/org/hawkular/metrics/core/jobs/JobsServiceImpl.java

```diff
@@ -102,7 +102,7 @@ public void setScheduler(Scheduler scheduler) {
 
         Boolean compressionEnabled = Boolean.valueOf(configuration.get(CompressData.ENABLED_CONFIG, "true"));
         if(compressionEnabled) {
-            CompressData compressDataJob = new CompressData(metricsService);
+            CompressData compressDataJob = new CompressData(metricsService, configurationService);
```

Comment:
This is now confusing.. we have two compress jobs enabled settings, doing different things.